index-pl.html

```diff
@@ -467,7 +467,7 @@ <h2>Jak Zatrzymać Epidemię</h2>
 
     <p>
       Prawdopodobnie słyszałaś/eś o terminie epidemiologicznym R0, czytane R-zero.
-      Jest to <a href="https://en.wikipedia.org/wiki/Basic_reproduction_number">liczba osób, które jedna zakażona osoba może zarazić</a> w populacji w której nie istnieje uprzednia odporność na daną chorobę.
+      Jest to <a href="https://en.wikipedia.org/wiki/Basic_reproduction_number">liczba osób, którą jedna zakażona osoba może zarazić</a> w populacji w której nie istnieje uprzednia odporność na daną chorobę.
```

Comment:
Jest to liczba osób jaką jedna zakażona osoba może zarazić w populacji, w której nie istnieje uprzednia odporność na daną chorobę.